isis/src/control/apps/findfeatures/main.cpp

```diff
@@ -1,377 +1,34 @@
-  // type.  Note that it was created as an image-to-image network. Must make
-  // adjustments if a ground network is requested.
-  if ( ui.WasEntered("ONET") ) {
-    ControlNet cnet;
-    cnet.SetNetworkId(ui.GetString("NETWORKID"));
-    cnet.SetUserName(Application::UserName());
-    cnet.SetDescription(best->matcher()->name());
-    cnet.SetCreatedDate(Application::DateTime());
-    QString target = ( ui.WasEntered("TARGET") ) ? ui.GetString("TARGET") :
-                                                   best->target();
-    cnet.SetTarget( target );
-    ID pointId( ui.GetString("POINTID"), ui.GetInteger("POINTINDEX") );
-    matcher.setDebugOff();
-    PvlGroup cnetinfo = matcher.network(cnet, *best, pointId);
-
-    if ( cnet.GetNumPoints() <= 0 ) {
-      QString mess = "No control points found!!";
-      logger->dbugout() << mess << "\n";
-      throw IException(IException::User, mess, _FILEINFO_);
-    }
-
-    // Umm..have to check this. Probably only makes sense with two images
-    if ( ui.GetString("NETTYPE").toLower() == "ground" ) {
-      cnetinfo += PvlKeyword("SpecialNetType", "Ground");
-      for ( int i = 0 ; i < cnet.GetNumPoints() ; i++ ) {
-        ControlPoint *point = cnet.GetPoint(i);
-        point->SetType(ControlPoint::Fixed);
-        point->Delete(matcher.query().id());
-        point->SetRefMeasure(0);
-      }
-    }
-
-    // Write out control network
-    cnet.Write( ui.GetFileName("ONET") );
-    Application::Log(cnetinfo);
-  }
-
-  // If user wants a list of matched images, write the list to the TOLIST filename
-  if ( ui.WasEntered("TOLIST") ) {
-    QLogger fout( QDebugLogger::create( ui.GetAsString("TOLIST"),
-                                             (QIODevice::WriteOnly |
-                                              QIODevice::Truncate) ) );
-    fout.logger() << matcher.query().name() << "\n";
-    MatcherSolution::MatchPairConstIterator mpair = best->begin();
-    while ( mpair != best->end() ) {
-      if ( mpair->size() > 0 ) {
-         fout.logger() << mpair->train().source().name() << "\n";
-      }
-      ++mpair;
-    }
-  }
-
-
-  // If user wants a list of failed matched images, write the list to the
-  // TONOTMATCHED file in append mode (assuming other runs will accumulate
-  // failed matches)
-  if ( ui.WasEntered("TONOTMATCHED") ) {
-    QLogger fout( QDebugLogger::create( ui.GetAsString("TONOTMATCHED"),
-                                             (QIODevice::WriteOnly |
-                                              QIODevice::Append) ) );
-    MatcherSolution::MatchPairConstIterator mpair = best->begin();
-    while ( mpair != best->end() ) {
-      if ( mpair->size() == 0 ) {
-         fout.logger() << mpair->train().source().name() << "\n";
-      }
-      ++mpair;
-    }
+  PvlGroup results = appLog.findGroup("Results");
+  if( ui.WasEntered("TO") && ui.IsInteractive() ) {
+    Application::GuiLog(results);
   }
 
-  return;
+  SessionLog::TheLog().AddResults(results);
```

Comment:
Out of curiosity where was this in the old code/what did it replace?

Reply:
The app used both logs and guilogs, so I tried to create a mix of the two templates.  This line comes from the [gui log section](https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#application-with-guilogs).